hera_sim/tests/test_vis.py

```diff
@@ -312,7 +313,7 @@ def align_src_to_healpix(point_source_pos, point_source_flux, nside=2 ** 4):
     )
```

Comment:
`astropy_healpix.lonlat_to_healpix` is preferred.

Reply:
And `astropy_healpix.nside_to_npix` is preferred for line 307 (which I can't comment on).